### DISCUSSION TITLE
[Miscellaneous] Ecosystem Apps API

### DISCUSSION THREAD

**Discussion by hanniabu** [ethereum-optimism/developers#145] [Miscellaneous]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

I was curious if there's an API to get the information listed on this page: https://www.optimism.io/apps/all

Or if the data is on github somewhere.

**Comment by sbvegan** [comment]
Hey @hanniabu, at the moment there is no api and unlikely there will be one in the future. The data is not publicly available on github right now either. I'm curious, what are you looking to use this data for?

**Comment by hanniabu** [comment]
I was going to use it to build an L2 app browser, but after more research I realized there's already resources like this.
